src/main/java/com/iridium/iridiumskyblock/shop/ShopItem.java

```diff
@@ -27,15 +28,17 @@
     /**
      * A short version of the default constructor.
      *
-     * @param name The name of the item
+     * @param name The name of the item in the shop
+     * @param displayName The name of the item when bought
      * @param type The material of the item
      * @param defaultAmount The amount of the item
      * @param slot The slot this item should be in
      * @param buyCost The cost for buying this item
      * @param sellReward The reward for selling this item
      */
-    public ShopItem(String name, XMaterial type, int defaultAmount, int slot, BuyCost buyCost, SellReward sellReward) {
+    public ShopItem(String name, String displayName, XMaterial type, int defaultAmount, int slot, BuyCost buyCost, SellReward sellReward) {
```

Comment:
You could also think about removing this displayName argument here and setting it to "" by default. It would make the shop config a lot cleaner